src/main/java/org/frc5687/infiniterecharge/robot/Constants.java

```diff
@@ -24,8 +24,8 @@
         public static final double ROTATION_SENSITIVITY = 0.75;
 
         public static final double CREEP_FACTOR = 0.25;
-        public static final double LEFT_DISTANCE_PER_PULSE = 0.0286206896551724;
-        public static final double RIGHT_DISTANCE_PER_PULSE = 0.0286206896551724;
+        public static final int CPR = 8192;
```

Comment:
What does CPR stand for?

Reply:
Counts Per Revolution, number is taken from Rev Robotics